services/pushover.py

```diff
@@ -14,19 +22,29 @@
 
 class PushoverError(Exception): pass
 
-def pushover(**kwargs):
```

Comment:
Any reason you changed this? I am no python expert but I do like `**kwargs` - nice and concise and easy to extend (as you are doing here).

Reply:
Yeah I'll have to think about this....probably could go back to how it worked now that its working....but when I started, this really threw me.  I'd not seen it before, maybe out the end of the method, but not the entire method definition.

Reply:
TBH I had to look up the docs as I hadn't used it myself before. I am a bit of a Python noob as well. But it does seem like a good way to handle these types of functions with plenty of options. Up to you.

Reply:
As mentioned, this broke my head when I tried switching back.  I'd probably need some guidance from you or @jpmens in how to reimplement it, if it's a better way of doing it.